libs/application/templates/parental-leave/README.md

```diff
@@ -1,5 +1,138 @@
+
+### API and X-Road
+
+All user interaction goes through our GraphQL API (`api`) which then handles communicating with other APIs.
+
+To communicate with VMST a request has to go through [X-Road](https://docs.devland.is/technical-overview/x-road). Both in development and on production environments.
+
+To connect to VMST test API you'll want to start the local proxy, following these steps:
+
+- Open AWS: https://island-is.awsapps.com/start#/
+  - island-is-development01 > AWSPowerUserAccess > Command line access
+- Choose an option you prefer in the modal
+- Connect to the proxy `./scripts/run-xroad-proxy.sh`
+
+See `libs/api/domains/directorate-of-labour/src/lib/directorate-of-labour.module.ts` for examples of VMST communication.
+
+### Contacts
+
+VMST API specific questions:
+[Guðmundur K. G. Kolka](mailto:gudmundur.kolka@advania.is), API and NAV developer
+
+VMST Devops:
+[Unnar Sigurðsson](mailto:unnar.sigurdsson@vmst.is), VMST API dev ops and x-road integration
+
+We also have two channels on the islandis slack #faedingarorlof-profanir and #faedingarorlof
+
+### Localisation
+
+All localisation can be found on Contentful.
+
+- [Parental leave application translations](https://app.contentful.com/spaces/8k0h54kbe6bj/entries/pl.application)
+- [Application system translations](https://app.contentful.com/spaces/8k0h54kbe6bj/entries/application.system)
+
+⚠️ When creating new text strings in the messages.ts file for the application, be sure to update Contentful, this is done using the command:
+
+```
+CONTENTFUL_MANAGEMENT_ACCESS_TOKEN={secret*} yarn nx run application-templates-parental-leave:extract-strings
+```
+
+_secret is found within AWS parameter store_
+
+See [link](https://docs.devland.is/apps/web/subpages#contentful) for further documentation for localisation in island.is repo.
+
+### Emails
+
+As previously mentioned, the application sends out emails to applicants and assignees. The email templates are stored in the parental leave api template module.
+
+When developing locally you’ll see preview links for would-be emails where you can see how they would be rendered and click links.
+
+## Setup
+
+See [application-system](https://docs.devland.is/apps/application-system) setup on how to get started.
+
+You'll want to start the `application-system-api` and `api` with these env variables to form the correct X-Road requests:
+
+- `XROAD_BASE_PATH_WITH_ENV=http://localhost:8081/r1/IS-DEV`
+- `XROAD_VMST_MEMBER_CODE=10003`
+- `XROAD_VMST_API_PATH=/VMST-ParentalLeave-Protected/ParentalLeaveApplication-v1`
+- `XROAD_CLIENT_ID=IS-DEV/GOV/10000/island-is-client`
+
+Then you'll also want these env variables:
+
+- `VMST_API_KEY`: to communicate with the VMST API, found in AWS parameter store
+- `CONTENTFUL_ACCESS_TOKEN`: to fetch translations, found in Contentful
```

Comment:
Would be nice to list commands to fetch these with [get-secrets](https://docs.devland.is/repository/aws-secrets#usage-to-fetch-secrets)